modules/order/src/Plugin/Field/FieldType/AdjustmentItem.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace Drupal\commerce_order\Plugin\Field\FieldType;
+
+use Drupal\commerce_order\Adjustment;
+use Drupal\commerce_price\Plugin\Field\FieldType\PriceItem;
+use Drupal\Core\Field\FieldStorageDefinitionInterface;
+use Drupal\Core\TypedData\DataDefinition;
+
+/**
+ * Plugin implementation of the 'commerce_price' field type.
+ *
+ * @todo See if bojanz throws bricks at me for extended PriceItem.
```

Comment:
There is nothing wrong with having specific columns for the adjustment, but I think it's more clear for AdjustmentItem to stand on its own, without people having to lookup PriceItem. We are talking about repeating 20 lines of code.